webapp/channels/src/components/user_settings/display/user_settings_theme/custom_theme_chooser/custom_theme_chooser.tsx

```diff
@@ -433,7 +433,7 @@ export default class CustomThemeChooser extends React.PureComponent<Props, State
                 />
                 <div className='mt-3'>
                     <button
-                        className='btn btn-link copy-theme-button'
```

Comment:
The `copy-theme-button` style from `webapp/channels/src/sass/routes/_settings.scss` can be removed.

Reply:
Done.